avogadro/io/cjsonformat.cpp

```diff
@@ -387,13 +398,23 @@ bool CjsonFormat::write(std::ostream& file, const Molecule& molecule)
     // Now get the MO matrix, potentially other things. Need to get a handle on
     // when we have just one (paired), or two (alpha and beta) to write.
     auto moMatrix = gaussian->moMatrix();
+    auto betaMatrix = gaussian->moMatrix(BasisSet::Beta);
     json moCoefficients;
-    for (int j = 0; j < moMatrix.cols(); ++j) {
-      for (int i = 0; i < moMatrix.rows(); ++i) {
+    for (int j = 0; j < moMatrix.cols(); ++j)
+      for (int i = 0; i < moMatrix.rows(); ++i)
         moCoefficients.push_back(moMatrix(i, j));
-      }
+
+    if (betaMatrix.cols() > 0 && betaMatrix.rows() > 0) {
+      json moBeta;
+      for (int j = 0; j < moMatrix.cols(); ++j)
+        for (int i = 0; i < moMatrix.rows(); ++i)
+          moBeta.push_back(moMatrix(i, j));
+
+      root["orbitals"]["alpha"] = moCoefficients;
```

Comment:
I think it should be clearer that these are coefficients. The key could also imply energies/eigenvalues.

How about "alphaCoeff" or "alphaCoefficients"

Reply:
Sounds good, this is pretty minor, I can update it.